config/nightwatch.js

```diff
@@ -1,14 +1,18 @@
 /* eslint-disable camelcase, strict */
 'use strict';
 
+const fs = require('fs-extra');
 const chromedriver = require('chromedriver');
 const seleniumServer = require('selenium-server');
 
 require('babel-register');
 require('babel-polyfill');
 
+const selenium_logs = './logs/selenium';
 const selenium_server_port = process.env.SELENIUM_PORT || 4444;
 
+fs.ensureDirSync(selenium_logs);
```

Comment:
I would much rather create the directory programmatically than check it into Git with `.keep` files.